.github/pull_request_template.md

```diff
@@ -1,31 +1,12 @@
-Fixes RNBW-####
-Figma link (if any):
+Fixes APP-####
 
 ## What changed (plus any additional context for devs)
 
 
-## Screen recordings / screenshots
```

Comment:
I think these 2 sections should stay as theyre not pertaining to the ticket but to the actual review, happy to discuss doe

I would like to delete everything in the final checklist and push that stuff to be more culturally top of mind